src/server/routes.pl

```diff
@@ -485,10 +485,17 @@
             param_value_search_message(Search, Message),
             param_value_search_graph_type(Search, Graph_Type),
             param_value_search_optional(Search, full_replace, boolean, false, Full_Replace),
+            param_value_search_optional(Search, json, boolean, false, JSON),
```

Comment:
It was only after reading `insert_document` that I realized this `JSON` that was being passed around was not a dictionary but a boolean. Can we use a different name here? `Is_JSON` or something else?